src/Symfony/Component/DependencyInjection/Compiler/AutowirePass.php

```diff
@@ -158,6 +163,52 @@ private function doProcessValue(mixed $value, bool $isRoot = false): mixed
         return $value;
     }
 
+    private function processAttribute(object $attribute, bool $isOptional = false): mixed
+    {
+        switch (true) {
```

Comment:
a `return match (true)` would do it, isn't it?

Reply:
Some of the cases have multiple lines, this isn't supported with `match`, is it?

Reply:
right